recipe_sha: b59583e0eddb8057c170e9dcffc245ec6114a8ea22e050c08697d3db06ffba70  # The hash of the recipe's meta.yaml at which this recipe failed to build.
skiplist: true # Set to true to skiplist this recipe so that it will be ignored as long as its latest commit is the one given above.
log: |2-


  Packaging mantis-msi
  Packaging mantis-msi-1.0.5-h5fa12a8_3

  WARNING: Importing conda-verify failed.  Please be sure to test your packages.  conda install conda-verify to make this message go away.
  number of files: 9
     INFO: sysroot: '/System/Library/Frameworks/' files: '['vmnet.framework/Versions/Current', 'vmnet.framework/Versions/A/_CodeSignature/CodeResources', 'vmnet.framework/Versions/A/Resources/version.plist', 'vmnet.framework/Versions/A/Resources/InfoPlist.loctable']'
     INFO: sysroot: '/usr/lib/' files: '['zsh/5.9/zsh/zselect.so', 'zsh/5.9/zsh/zpty.so', 'zsh/5.9/zsh/zprof.so', 'zsh/5.9/zsh/zleparameter.so']'
     INFO: sysroot: '/opt/X11/' files: '[]'
     INFO (mantis-msi,bin/RepeatFinder): Needed DSO /System/Library/Frameworks/libc.1.dylib found in $SYSROOT
     INFO (mantis-msi,bin/RepeatFinder): Needed DSO /System/Library/Frameworks/libSystem.B.dylib found in $SYSROOT
  WARNING (mantis-msi): dso library package conda-forge/osx-64::libcxx==19.1.6=hf95d169_1 in requirements/run but it is not used (i.e. it is overdepending or perhaps statically linked? If that is what you want then add it to build/ignore_run_exports)
  Fixing permissions
  INFO :: Time taken to mark (prefix)
          0 replacements in 0 files was 0.04 seconds
  TEST START: /opt/mambaforge/envs/bioconda/conda-bld/osx-64/mantis-msi-1.0.5-h5fa12a8_3.tar.bz2
  Renaming work directory '/opt/mambaforge/envs/bioconda/conda-bld/mantis-msi_1734565582638/work' to '/opt/mambaforge/envs/bioconda/conda-bld/mantis-msi_1734565582638/work_moved_mantis-msi-1.0.5-h5fa12a8_3_osx-64'
  shutil.move(work)=/opt/mambaforge/envs/bioconda/conda-bld/mantis-msi_1734565582638/work, dest=/opt/mambaforge/envs/bioconda/conda-bld/mantis-msi_1734565582638/work_moved_mantis-msi-1.0.5-h5fa12a8_3_osx-64)
  Reloading output folder (local): ...working... done
  Solving environment (_test_env): ...working... done

  ## Package Plan ##

    environment location: /opt/mambaforge/envs/bioconda/conda-bld/mantis-msi_1734565582638/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol


  The following NEW packages will be INSTALLED:

      bzip2:           1.0.8-hfdf4475_7          conda-forge
      c-ares:          1.34.4-hf13058a_0         conda-forge
      ca-certificates: 2024.12.14-h8857fd0_0     conda-forge
      krb5:            1.21.3-h37d8d59_0         conda-forge
      libblas:         3.9.0-26_osx64_openblas   conda-forge
      libcblas:        3.9.0-26_osx64_openblas   conda-forge
      libcurl:         8.11.1-h5dec5d8_0         conda-forge
      libcxx:          19.1.6-hf95d169_1         conda-forge
      libdeflate:      1.22-h00291cd_0           conda-forge
      libedit:         3.1.20191231-h0678c8f_2   conda-forge
      libev:           4.33-h10d778d_2           conda-forge
      libexpat:        2.6.4-h240833e_0          conda-forge
      libffi:          3.4.2-h0d85af4_5          conda-forge
      libgfortran:     5.0.0-13_2_0_h97931a8_3   conda-forge
      libgfortran5:    13.2.0-h2873a65_3         conda-forge
      liblapack:       3.9.0-26_osx64_openblas   conda-forge
      liblzma:         5.6.3-hd471939_1          conda-forge
      libnghttp2:      1.64.0-hc7306c3_0         conda-forge
      libopenblas:     0.3.28-openmp_hbf64a52_1  conda-forge
      libsqlite:       3.47.2-hdb6dae5_0         conda-forge
      libssh2:         1.11.1-h3dc7d44_0         conda-forge
      libzlib:         1.3.1-hd23fc13_2          conda-forge
      llvm-openmp:     19.1.6-ha54dae1_0         conda-forge
      mantis-msi:      1.0.5-h5fa12a8_3          local
      ncurses:         6.5-hf036a51_1            conda-forge
      numpy:           2.2.0-py312h22eab8f_0     conda-forge
      openssl:         3.4.0-hd471939_0          conda-forge
      pysam:           0.22.1-py312hc68ed8a_3    bioconda
      python:          3.12.8-h9ccd52b_1_cpython conda-forge
      python_abi:      3.12-5_cp312              conda-forge
      readline:        8.2-h9e318b2_1            conda-forge
      tk:              8.6.13-h1abcd95_1         conda-forge
      tzdata:          2024b-hc8b5060_0          conda-forge
      zstd:            1.5.6-h915ae27_0          conda-forge

  Preparing transaction: ...working... done
  Verifying transaction: ...working... done
  Executing transaction: ...working... done
   mantis-msi.py --help
  WARNING: Tests failed for mantis-msi-1.0.5-h5fa12a8_3.tar.bz2 - moving package to /opt/mambaforge/envs/bioconda/conda-bld/broken
  Traceback (most recent call last):
    File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/build.py", line 3523, in test
  export PREFIX=/opt/mambaforge/envs/bioconda/conda-bld/mantis-msi_1734565582638/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehol
  export SRC_DIR=/opt/mambaforge/envs/bioconda/conda-bld/mantis-msi_1734565582638/test_tmp
  Microsatellite Analysis for Normal-Tumor InStability (v1.0.4)
  Error: You must have Pysam available in your environment!
  Error: You must have NumPy available in your environment!
  Please check your $PYTHONPATH to make sure you have properly included required moudles/libraries in it.
      utils.check_call_env(
    File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/utils.py", line 404, in check_call_env
      return _func_defaulting_env_to_os_environ("call", *popenargs, **kwargs)
    File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/utils.py", line 380, in _func_defaulting_env_to_os_environ
      raise subprocess.CalledProcessError(proc.returncode, _args)
  subprocess.CalledProcessError: Command '['/bin/bash', '-o', 'errexit', '/opt/mambaforge/envs/bioconda/conda-bld/mantis-msi_1734565582638/test_tmp/conda_test_runner.sh']' returned non-zero exit status 1.

  During handling of the above exception, another exception occurred:

  Traceback (most recent call last):
    File "/opt/mambaforge/envs/bioconda/bin/conda-build", line 11, in <module>
      sys.exit(execute())
    File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/cli/main_build.py", line 589, in execute
      api.build(
    File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/api.py", line 209, in build
      return build_tree(
    File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/build.py", line 3727, in build_tree
      test(pkg, config=metadata.config.copy(), stats=stats)
    File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/build.py", line 3537, in test
      tests_failed(
    File "/opt/mambaforge/envs/bioconda/lib/python3.10/site-packages/conda_build/build.py", line 3584, in tests_failed
      raise CondaBuildUserError("TESTS FAILED: "  os.path.basename(pkg))
  conda_build.exceptions.CondaBuildUserError: TESTS FAILED: mantis-msi-1.0.5-h5fa12a8_3.tar.bz2
# Last 100 lines of the build log.
category: |-
  test failure
